config/autoload/routes.global.php

```diff
@@ -147,6 +150,33 @@
             'allowed_methods' => ['GET', 'POST'],
         ],
         [
+            'name' => 'account-locations-list',
+            'path' => '/account/locations',
+            'middleware' => [
+                App\Middleware\Authentication::class,
+                App\Action\Account\Location\ListLocationsAction::class,
+            ],
+            'allowed_methods' => ['GET'],
+        ],
+        [
+            'name' => 'account-location-add',
+            'path' => '/account/location/add',
+            'middleware' => [
+                App\Middleware\Authentication::class,
+                App\Action\Account\Location\AddLocationAction::class,
+            ],
+            'allowed_methods' => ['GET', 'POST'],
+        ],
+        [
+            'name' => 'account-location-edit',
+            'path' => '/account/location/{uuid}/edit',
```

Comment:
The only way you'll get that is if you manually type a URL in, so IMO it's an acceptable situation (the 500). If you're following links, everything should be fine.